genesyscloud/telephony_providers_edges_trunk/genesyscloud_telephony_providers_edges_trunk_init_test.go

```diff
@@ -3,7 +3,7 @@ package telephony_providers_edges_trunk
 import (
 	"sync"
 	"terraform-provider-genesyscloud/genesyscloud/location"
-	telephony "terraform-provider-genesyscloud/genesyscloud/telephony"
+	telephony "terraform-provider-genesyscloud/genesyscloud/telephony_provider_edges_trunkbasesettings"
```

Comment:
Do you want to rename telephony to trunkbasesettings?

Reply:
Yes